docs/devops.md

```diff
@@ -148,7 +148,7 @@ And that's it, this will automatically trigger a build on the build pipeline for
 
 Here are some additional steps that need to be followed by a freeCodeCamp.org Staff developer. To prevent any accidental pushes we have a couple of manual approval steps configured on the pipelines.
 
-Once a build artifact is ready on the `production-current` branch, it will trigger a release on the release pipeline. 
+Once a build artifact is ready on the `production-current` branch, it will trigger a release on the release pipeline.
 
 Next, freeCodeCamp.org developer staff team will receive an email. They can either *approve* or *reject* the release. Approval or rejection depends, if changes were nicely working and tested on the staging application. Each approval lasts only for 4 hours to avoid queuing up. Post the limit it gets auto rejected, wherein a staff will re-trigger the release pipeline manually.
```

Comment:
```suggestion
Next, the freeCodeCamp.org developer staff team will receive an email. They can either *approve* or *reject* the release. If the changes are working nicely and have been tested on the staging application, then it can be approved. This must happen within 4 hours of the release being triggered or it will automatically get rejected.  If this happens a staff member will need to re-trigger the release pipeline manually.
```

Reply:
Sorry if this is a little premature.  I realise it's still in draft.

Reply:
Well, this PR is mostly targeting the VM provisioning etc, and it will be a while before I can get back to it. It would be awesome if you could make a PR with the suggested changes ?

Reply:
Sure thing.

Reply:
Weird, I can't dismiss my review.

Reply:
Probably because its draft. IDK either.